fs/fs.go

```diff
@@ -58,12 +60,15 @@ type RealFsInfo struct {
 	labels map[string]string
 
 	dmsetup dmsetupClient
+
+	thinPoolWatcher *volume.ThinPoolWatcher
```

Comment:
@vishh I do not understand the role of the FsInfo abstraction vs. The FsHandler one -- could you clarify, perhaps?